ant-chapters/chapter1.tex

```diff
@@ -1,5 +1,103 @@
+
+\section{Motivation}
+A natural way to begin a book about Algebraic Number Theory is by introducing number fields. While we will define them much more precisely and generally soon,
+we may temporarily think of them as fields of rational numbers extended by ``external" elements. Rings of integers are similar, but are based on $\Z$ instead.
+
+\begin{ex} A typical ring of integers is $\Z[\sqrt{2}] = \left\{m + n \sqrt{2} : m,n \in \Z\right\}$.
+\end{ex}
+
+\noindent This construction is very powerful, and can be used to solve many classical problems, such as:
+\begin{itemize}
+\item \textbf{Diophantine Equations}: For example, solving the Diophantine equation $x^2 - dy^2 = 1$ (Pell's equation) can be understood as a problem in $\Z(\sqrt{d})$. Without going into detail for now, norm of $x + y\sqrt{d}$ in such ring is defined as $x^2 - dy^2$, which corresponds to equation we're interested in.
+\item \textbf{Pythagorean Triples}: In the ring of $\Z[i]$ norm of $a + bi$ is defined as $a^2 + b^2$. This can be used to study integral solutions to $a^2 + b^2 = c^2$ (i.e. Pythagorean Triples).
+\item \textbf{Special cases of Fermat's Last Theorem}: For instance, proving that there are no integer solutions to $x^3 + y^3 = z^3$ can be stated using the ring of integers $\Z(\omega)$ (where $\omega$ is a cube root of unity).
+\end{itemize}
+
+\noindent In the following sections we'll learn about elements of these rings, and their properties.
+
+\section{Algebraic Numbers}
+\begin{df}
+\textbf{Algebraic numbers} are complex numbers that are roots of a polynomial with coefficients in $\Q$. The set of algebraic numbers is denoted $\overline{\Q}$.
+\end{df}
+
+\begin{ex} Examples of algebraic numbers include:
+\begin{itemize}
+\item $1$ is a root of $1-x$ and $3x-3$.
+\item $\frac{1}{2}$ is a root of $2x - 1$ and $x - \frac{1}{2}$.
+\item $\sqrt{2}$ is a root of $x^2 - 2$ and $x^5 + 2x^4 - 2x^3 - 4x^2$.
+\item $i$ is a root of $x^2 + 1$.
+\end{itemize}
+\end{ex}
+
+\begin{df}
+A number is algebraic if any such polynomial exist. Since we're working in $\Q$ we can scale the polynomial to be monic (i.e. with 1 as a leading coefficient). The monic polynomial with a smallest degree is called a \textbf{minimal polynomial} over $\Q$.
+\end{df}
+
+\begin{df}
+The \textbf{degree} of algebraic number $\alpha$ is defined as the degree of its minimal polynomial.
+\end{df}
+
+\begin{ex} Examples of minimal polynomials:
+\begin{itemize}
+\item The minimal polynomial of $1$ is $x-1$. So $\deg 1 = 1$.
+\item The minimal polynomial of $\frac{1}{2}$ is $x - \frac{1}{2}$. So $\deg \frac{1}{2} = 1$.
+\item The minimal polynomial of $\sqrt{2}$ is $x^2 - 2$. So $\deg \sqrt{2} = 2$.
+\item The minimal polynomial of $i$ is $x^2 + 1$. So $\deg i = 2$.
+\item The minimal polynomial of $\sqrt[3]{2}$ is $x^3 - 2$. So $\deg \sqrt[3]{2} = 3$.

```

Comment:
These are the long and frequent examples I've mentioned in the description. I may be overdoing this here with 5 examples of minimal polynomials, but on the other hand I don't see how it can hurt.